### ISSUE TITLE
Does not work on Android

### ISSUE THREAD

**Issue by rugk** [rugk/how-did-i-get-here#4]
For some reason it does not work on Android. It never seems to get the tab list.

**Comment by linusbrogan** [comment]
Firefox for Android does not include the necessary APIs for extensions (`Tab.favIconUrl`, `Tab.openerTabId`, `sessions.getTabValue`). See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs for supported APIs.

**Comment by rugk** [comment]
Yeah, particularly `openerTabId` is critical thanks. This add-on IIRC has some fallback, but yeah…
It's a problem at least.
